spec/dataframe_spec.rb

```diff
@@ -3669,15 +3669,15 @@ def create_test(*args, &_proc)
       expect(df_union.index.to_a).to eq v1 + v2
     end
   end
-  
+
   context '#inspect' do
     subject { df.inspect }
 
     context 'empty' do
       let(:df) { Daru::DataFrame.new({}, order: %w[a b c])}
       it { is_expected.to eq %Q{
         |#<Daru::DataFrame(0x3)>
-        |   a   b   c
+        |       a   b   c
```

Comment:
Well, I can guess it now makes a space for an (absent) index name... OK, let it be so.